src/lint.py

```diff
@@ -210,6 +210,18 @@ def __init__(self, cursor: Cursor):
             for arg in cursor.get_arguments()
         ]
 
+    def check_ownership(self) -> None:
+        """
+        Check if a function returns RzList pointer and if it does,
+        verify if the function has either RZ_OWN or RZ_BORROW annotation.
+        If not, add a warning.
+        """
+        if self.name.startswith('RzList*') and not ('RZ_OWN' in self.annotations or 'RZ_BORROW' in self.annotations):
```

Comment:
probably you should check if the return ends with `*` so we know it is a pointer.

Reply:
Right, also comparing with `RzList*` will never match, because we use the one with the space instead in our syntax.